exercises/collatz-conjecture/example.js

```diff
@@ -0,0 +1,20 @@
+'use strict';
+
+function collatzConjecture() {}
+
+collatzConjecture.prototype.steps = function(no){
```

Comment:
May I suggest that you could use the full word `number` as a variable name?  It would make this immediately more understandable.